database/src/migrations/20250527114701_update_survey_methods.ts

```diff
@@ -0,0 +1,87 @@
+import { Knex } from 'knex';
+
+/**
+ * Adding more values to method lookup table as discussed in meeting about getting existing SPI values into SIMS.
+ * @export
+ * @param {Knex} knex
+ * @return {*}  {Promise<void>}
+ */
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`--sql
+
+  SET SEARCH_PATH=biohub;
+
+  ----------------------------------------------------------------------------------------
+  --Deleting previous rows that no longer apply to the method_lookup table
+  ----------------------------------------------------------------------------------------
+    DELETE FROM method_lookup
```

Comment:
We should probably change any existing Aerial transects, ground transects, aquatic transects, and underwater transects to "Visual observation". If these transect methods exist in the survey_sample_method table, trying to drop them from the method_lookup table will fail with a foreign key constraint error.

Try adding this to the end of this migration, after having created the Visual encounter method if it doesn't already exist, followed by the deletion of the transect methods part.

```
UPDATE survey_sample_method
SET method_lookup_id = (SELECT method_lookup_id FROM method_lookup WHERE name = 'Visual encounter')
WHERE method_lookup_id = (SELECT method_lookup_id FROM method_lookup WHERE name IN ('Aerial transect', 'Ground transect', 'Aquatic transect', 'Underwater transect');
```

Reply:
Ive had a go at this and recommite. All looks good to me